praw/models/__init__.py

```diff
@@ -15,15 +15,16 @@
 from .reddit.multi import Multireddit  # NOQA
 from .reddit.redditor import Redditor  # NOQA
 from .reddit.submission import Submission  # NOQA
-from .reddit.subreddit import Subreddit  # NOQA
+from .reddit.subreddit import ModmailAction, ModmailConversation, ModmailMessage, Subreddit  # NOQA
```

Comment:
What do you think about creating a new module `reddit.modmail` to put these classes in?